lib/Refactoring/Refactoring.cpp

```diff
@@ -8946,15 +8954,19 @@ void swift::ide::collectAvailableRefactorings(
 
   bool enableInternalRefactoring = getenv("SWIFT_ENABLE_INTERNAL_REFACTORING_ACTIONS");
 
+  SmallVector<RefactorAvailabilityInfo, 8> Infos;
```

Comment:
I'm not a fan of returning `SmallVector`. When returning this, is this copying the vector or return value optimized because `SmallVector< RefactorAvailabilityInfo>` happens to be the same as `SmallVector< RefactorAvailabilityInfo, 8>`?

Reply:
I've changed them all to `SmallVector<RefactorAvailabilityInfo, 0>`. Passing them in would be slightly better, but IMO it isn't worth the readability loss in this case.